master_duff.py

```diff
@@ -174,8 +187,10 @@ async def produceStats():
                 # Will be "Game Removed" for Undo records
                 record_title = elo_info["title"]
 
+                # CHANGE FIELDS TO DESCRIPTION IN V2
                 # Text of first embed, used to determine failed records
-                first_item_name = elo_info["fields"][0]["name"]
+                # first_item_name = elo_info["fields"][0]["name"]
+                first_item_name = elo_info["description"]
             except Exception:
```

Comment:
One thing that will help in debugging is to just remove any `try` and `except` blocks that are being used to catch errors or edge cases.  I'm not sure if the line above has an error or not, but if so, it will invalidate the `record_title` right below here and if `record_title` is empty than the logic on line 216 won't work

That said, I'm pretty sure the note above is what is actually causing the issue you described, the check that requires the `elo_info` dictionary to have a key `fields`